src/store/stats.js

```diff
@@ -0,0 +1,70 @@
+import LocalStore from 'electron-store';
+import { Connection, PublicKey } from '@solana/web3.js';
+import { action, observable, flow } from 'mobx';
+import url from '../url';
+import Replicator from '../replicator';
+import AppStore from './app';
+
+const DEFAULT_LAMPORTS = 10000;
+export const localStore = new LocalStore();
+
+class StatsStore {
+  constructor() {
+    this.connection = new Connection(url);
+    this.replicator = new Replicator(this.connection);
+  }
+
+  @observable stats = {
+    transactionCount: 0,
+    totalMined: localStore.get('totalMined', 0),
+    newMined: 0,
+    totalSupply: 0,
+    depositMinimumLamports: localStore.get(
+      'depositMinimumLamports',
+      DEFAULT_LAMPORTS
```

Comment:
IMO just inline the `10000` here.  That `DEFAULT_LAMPORTS` constant isn't doing much

Reply:
no magic numbers :) next pr i will move this to other constants to constants.js file. stay tuned!